client/me/help/help-contact/index.jsx

```diff
@@ -595,12 +597,16 @@ export default connect(
 	state => {
 		const helpSelectedSiteId = getHelpSelectedSiteId( state );
 		const selectedSitePlan = getSitePlan( state, helpSelectedSiteId );
+		const localizedLanguageNames = getLocalizedLanguageNames( state );
 		return {
 			currentUserLocale: getCurrentUserLocale( state ),
 			currentUser: getCurrentUser( state ),
 			getUserInfo: getHappychatUserInfo( state ),
 			hasHappychatLocalizedSupport: hasHappychatLocalizedSupport( state ),
 			hasAskedADirectlyQuestion: hasUserAskedADirectlyQuestion( state ),
+			defaultLanguage: localizedLanguageNames
+				? localizedLanguageNames[ config( 'i18n_default_locale_slug' ) ].localized
+				: localizedLanguageNames,
```

Comment:
`localizedLanguageNames` could be `null` before receiving the full list. During that short amount of time, it would show `en` like this in French:
<img width="749" alt="2018-04-17 12 17 15" src="https://user-images.githubusercontent.com/1842898/38848514-08223f58-423a-11e8-812e-ce4e733074db.png">

maybe we could just show the translated "chat with us" when `localizedLanguageNames` is not there yet.

Also, I'd suggest us to not complicate a connected props too much. In this case, it would take one extra `config()` look up for the non-changing configuration value, `i18n_default_locale_slug`, and some extra inferring to understand `localizedLanguageNames` can be null and the ternary expression will still be null in that case. Could we consider to keep `defaultLanguage` simple like it was and move the logic of localizing the default language name right before overriding the button label? It's just a personal preference though, so please feel free to keep it :)

Reply:
Thanks for testing this southp!

> maybe we could just show the translated "chat with us" when localizedLanguageNames is not there yet.

Actually, that sounds like the easiest option. 

We already have a sentence above the buttons explaining that chat is in English. 

We could just remove 'English from the button and therefore the need to poll the languagenames API at all, removing all the complications @southp is talking about.

@cecilerainon What do you think?

Reply:
Since that note was added, we've had fewer users contacting us in their native language, but we still have users contacting us who miss that note and expect a reply in their language.

Here with a button, they can't miss it because they're actually clicking it to initiate the chat, so I'd be in favor of adding **Chat with us** ***in English*** if that's not too much trouble, please. :)